src/app/app-modules/pharmacist/redir-fallback/redir-fallback.component.html

```
@@ -1,5 +1,3 @@
-<div class="overlay">
-  <div class="overlay-content">
-    <mat-spinner></mat-spinner>
-  </div>
+<div class="fixed inset-0 z-[1001] flex items-center justify-center bg-black/30">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)